common/formatting/token_partition_tree_test.cc

```diff
@@ -398,4 +398,486 @@ TEST_F(MoveLastLeafIntoPreviousSiblingTest, TwoGenerations) {
                                     << *diff.right << '\n';
 }
 
+class TokenPartitionTreeWrapperTest
+    : public TokenPartitionTreeTestFixture {};
+
+TEST_F(TokenPartitionTreeWrapperTest, NonGrouping) {
+  const auto& preformat_tokens = pre_format_tokens_;
+  const auto begin = preformat_tokens.begin();
+
+  UnwrappedLine one(0, begin);
+  one.SpanUpToToken(begin + 1);
+
+  using tree_type = TokenPartitionTree;
+  tree_type tree{
+    one,
+  };
+
+  TokenPartitionTreeWrapper tree_wrapper(tree);
+
+  const auto& tree_range = tree.Value().TokensRange();
+  const auto& tree_wrapper_range = tree_wrapper.Value().TokensRange();
+  EXPECT_TRUE(tree_range.begin() == tree_wrapper_range.begin());
+  EXPECT_TRUE(tree_range.end() == tree_wrapper_range.end());
+}
+
+TEST_F(TokenPartitionTreeWrapperTest, OneGrouping) {
+  const auto& preformat_tokens = pre_format_tokens_;
+  const auto begin = preformat_tokens.begin();
+
+  UnwrappedLine one(0, begin);
+  one.SpanUpToToken(begin + 1);
+
+  using tree_type = TokenPartitionTree;
+  tree_type tree{
+    one,
+  };
+
+  VectorTree<TokenPartitionTreeWrapper> tree_wrapper(tree.Value());
+  auto* group = tree_wrapper.NewChild(tree.Value());
+  auto* child = group->NewChild(tree.Children()[0]);
+
+  // FIXME: check structure
+
+  EXPECT_EQ(tree_wrapper.Children()[0].Children()[0].Value().Node(), &tree.Children()[0]);
+
+  const auto& tree_range = tree.Value().TokensRange();
+  const auto& tree_wrapper_range = tree_wrapper.Value().Value().TokensRange();
+  EXPECT_TRUE(tree_range.begin() == tree_wrapper_range.begin());
+  EXPECT_TRUE(tree_range.end() == tree_wrapper_range.end());
```

Comment:
Does `EXPECT_EQ` work here?

Reply:
Outdated. But using `EXPECT_EQ` in other tests.